src/uris/deepLinks/deepLinkService.ts

```diff
@@ -1107,12 +1112,17 @@ export class DeepLinkService implements Disposable {
 								this._context.action === DeepLinkActionType.SwitchToPullRequestWorktree ||
 								this._context.action === DeepLinkActionType.SwitchToAndSuggestPullRequest
 							) {
-								action = DeepLinkServiceAction.OpenSwitch;
+								action = this._context.openedInNewWindow
+									? DeepLinkServiceAction.OpenAllPrChanges
+									: DeepLinkServiceAction.OpenSwitch;
 							} else {
 								action = DeepLinkServiceAction.OpenGraph;
 							}
 							break;
 					}
+					window.showInformationMessage(
+						`go to target: ${targetType} - ${this._context.action} - ${this._context.openedInNewWindow} -/ ${action}`,
+					);
```

Comment:
This extra notification is unnecessary.

Reply:
I left it here for you to see why couldn't you test it. I'll remove